_extras/guide.md

```diff
@@ -11,12 +11,12 @@ encounter setup issues, please file an issue with the tags 'High-priority'.
 
 ## Checking installations.
 
-In the `_include/scripts` directory, you will find a script called check_env.py This checks the
+In the [`_includes/scripts`](../_includes/scripts) directory, you will find a script called check_env.py This checks the
```

Comment:
I'm not sure this link will work in the lesson site after the page has been built and rendered - I think it will end up pointing to https://datacarpentry.org/python-ecology-lesson/_includes/scripts, which does not exist.

Anyway, I think the link target is best set as the full URL to the directory on GitHub, i.e. "https://github.com/datacarpentry/python-ecology-lesson/tree/gh-pages/_includes/scripts" because this is most reliable way for someone to find it when teaching a workshop. 

If they are working offline, they will need to have downloaded the repository and built their own local version. In which case I suspect they are likely to also be able to find the local equivalent location of the script.